tests/installation/confirm_installation_maintenance.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module confirms the popup for installation and additionally the license popup in case it appears.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+
+sub run {
+    my $install_popup = $testapi::distri->get_ok_popup_controller();
+    my $license_popup = $testapi::distri->get_license_popup();
+    # First check is for the confirm installation popup in order to save time as
+    # license popup will rarely appear.
+    if ($install_popup->get_ok_popup()) {
```

Comment:
but this method `get_ok_pop` will die if doesn't find it, so you need to create a new one:
`is_ok_popup_visible()` which return 1 or 0.